README.md

```diff
@@ -2,6 +2,13 @@
 
 The goal of *starfish* is to **prototype** a reference pipeline for the analysis of image-based transcriptomics data that works across technologies. This is a **work in progress** and will be developed in the open.
 
+We are currently in **pre-alpha**, finishing proof of concept pipelines for each of the spaceTx contributors that leverage starfish's shared object model.
+At this time starfish is mature enough to support computational developers interested in adapting other assays to starfish's object model.
+However, please note that these endeavors, while strongly encouraged, will likely come with significant computational time investment.
+
+Please see our project [roadmap](https://docs.google.com/document/d/1ZEzH42o7mn8ukSwONmMJpNQxoctWYHSMwVos-ZthyhI/edit?usp=sharing) for a more detailed view of our development priorities.
```

Comment:
this roadmap is so spacetx specific and vague (e.g., what's a call 1) that I don't think we should link here. Rather, the table of assays we currently support and intend to support is probably better to have?